src/Catty/Helpers/Util/Util.m

```diff
@@ -222,7 +222,7 @@ + (void)askUserForUniqueNameAndPerformAction:(SEL)action
                               minInputLength:(NSUInteger)minInputLength
                               maxInputLength:(NSUInteger)maxInputLength
                     invalidInputAlertMessage:(NSString*)invalidInputAlertMessage
-                               existingNames:(NSArray*)existingNames {
+                               existingNames:(NSOrderedSet*)existingNames {
```

Comment:
Is there a reason why you changed this parameter? This has not been changed in the interface and will probably lead to some problems, please revert this change.

Reply:
Oh this was my fault. I don't remember why I changed that. I changed it back.